client/i18n/locales/english/intro.json

```
@@ -3729,7 +3735,10 @@
       "kegi": { "title": "377", "intro": [] },
       "vnfu": { "title": "379", "intro": [] },
       "rlli": { "title": "380", "intro": [] },
-      "rjpv": { "title": "381", "intro": [] },
+      "In this lab, you will use object oriented programming to calcute the areas of different polygons like squares and rectangles.rjpv": {
+        "title": "381",
+        "intro": []
+      },
```

```suggestion
      "rjpv": { "title": "381", "intro": [] },
```

looks like a mistake here?